recipes/r-rcarbon/meta.yaml

```
@@ -0,0 +1,104 @@
+{% set version = '1.5.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rcarbon
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rcarbon_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rcarbon/rcarbon_{{ version }}.tar.gz
+  sha256: f59d4fec1c47fde1da7037273a3acc07079983280059aac23e8ac7a2a204ee71
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-dosnow
+    - r-foreach
+    - r-iterators
+    - r-knitr
+    - r-sf
+    - r-snow
+    - r-spatstat >=2.0_0
+    - r-spatstat.explore
+    - r-spatstat.geom
+    - r-spatstat.linnet
+    - r-spatstat.model
+  run:
+    - r-base
+    - r-dosnow
+    - r-foreach
+    - r-iterators
+    - r-knitr
+    - r-sf
+    - r-snow
+    - r-spatstat >=2.0_0
+    - r-spatstat.explore
+    - r-spatstat.geom
+    - r-spatstat.linnet
+    - r-spatstat.model
+
+test:
+  commands:
+    - $R -e "library('rcarbon')"           # [not win]
+    - "\"%R%\" -e \"library('rcarbon')\""  # [win]
+
+about:
+  home: https://github.com/ahb108/rcarbon/
+  license: GPL-2.0-only
```

DESCRIPTION shows `License: GPL (>= 2)`, so

```suggestion
  license: GPL-2.0-or-later
```